Custom/annatar.yml

```diff
@@ -78,13 +67,14 @@ search:
   rows:
     selector: streams
     missingAttributeEqualsNoResults: true
-
   fields:
     title:
       selector: title
       filters:
         - name: split
           args: ["\n", 0]
+    details:
+      selector: url
```

Comment:
why is this important?

Reply:
To fetch the url

Reply:
Things is this will still give an error on automatic Season Searches as the endpoint expects an episode. But as soon as Bret fixes that this should also support Season Packs just fine.